config/renovate/renovate.json

```diff
@@ -98,7 +98,13 @@
     ],
     "schedule": [
       "after 5am on saturday"
-    ]
+    ],
+    "postUpgradeTasks": {
```

Comment:
This is the global config, and this command will be applied to all components on-boarded to Konflux. Is this intended to happen for every component? If not, putting this in a local config would be better, otherwise I'm ok with keeping this.

Reply:
Yes, the migration tool is intended to run for all components.